Makefile

```diff
@@ -40,7 +40,17 @@ install: build
 	@echo "    }"
 	@echo "  }"
 	@echo "}"
-	
+
+install-tf-0.12: build
+	@echo "✓ Installing provider for Terraform 0.12 ..."
+	@mkdir -p '$(HOME)/.terraform.d/plugins/$(shell go version | awk '{print $$4}' | sed 's#/#_#')/'
+	@cp terraform-provider-databricks '$(HOME)/.terraform.d/plugins/$(shell go version | awk '{print $$4}' | sed 's#/#_#')/terraform-provider-databricks_v$(shell ./terraform-provider-databricks version)'
+	@echo "✓ Use the following configuration for Terraform 0.12 to enable the version you've built"
+	@echo ""
+	@echo 'provider "databricks" {'
+	@echo '  version = "0.3.0"'
```

Comment:
version pinning is not really needed for 0.12, 🤷 . it's going against installer script we provide

Reply:
with current `make install` it doesn't fine provider, unfortunately...

Reply:
Even after removing ~/.terraform.d ?...

Reply:
Ok, it works if I do install like our install script, and remove the version.  I'll remove this target completely...